test/unit/BondingManager.js

```diff
@@ -412,15 +417,14 @@ contract("BondingManager", accounts => {
             })
 
             it("should fire a Bond event when bonding from unbonded", async () => {
-                bondingManager.Bond({}).on("data", e => {
-                    assert.equal(e.returnValues.newDelegate, transcoder0, "wrong newDelegate in Bond event")
-                    assert.equal(e.returnValues.oldDelegate, constants.NULL_ADDRESS, "wrong oldDelegate in Bond event")
-                    assert.equal(e.returnValues.delegator, delegator, "wrong delegator in Bond event")
-                    assert.equal(e.returnValues.additionalAmount, 1000, "wrong additionalAmount in Bond event")
-                    assert.equal(e.returnValues.bondedAmount, 1000, "wrong bondedAmount in Bond event")
-                })
-
-                await bondingManager.bond(1000, transcoder0, {from: delegator})
+                const txRes = await bondingManager.bond(1000, transcoder0, {from: delegator})
+                truffleAssert.eventEmitted(
+                    txRes,
+                    "Bond",
+                    e => e.newDelegate == transcoder0 && e.oldDelegate == constants.NULL_ADDRESS &&
```

Comment:
When we have many successive checks like this (i.e. > 3-4) I feel like having each check on a separate line can improve readability. 

Ex.

(Indentation could be updated)

```
truffleAssert.eventEmitted(txResult, "EarningsClaimed", e => {
    return e.delegate === transcoder &&
            e.delegator == delegator1 &&
            e.fees == expFees.toString() &&
            e.rewards == expRewards.toString() &&
            e.startRound == (currentRound + 1).toString() &&
            e.endRound == (currentRound + 1).toString()
})
```

WDYT?

Reply:
Yeah that might be a little neater indeed

Reply:
fixed in ad3cb40